### ISSUE TITLE
Use QFileSystemWatcher for opened Editors

### ISSUE THREAD

**Issue by SimLV** [dail8859/NotepadNext#558]
### Description

Subj

**Comment by dail8859** [comment]
I agree QFileSystemWatcher is a possible solution. I've not tried it myself but I'm curious how to properly handle changes to the file that are expected, such as when the file get's saved by the application. But this is probably one of those things that has to be tested to see how it behaves.
